tests/cli/test_cli.py

```diff
@@ -2308,6 +2308,11 @@ def test_edit(ctx):
     _run_command([cl, 'edit', uuid, '-f', 'request_memory', 'invalid_value'], expected_exit_code=1)
 
 
+@TestModule.register('work')
+def test(ctx):
+    uuid = _run_command([cl, 'work', 'stanford::'], expected_exit_code=1)
```

Comment:
```suggestion
    uuid = _run_command([cl, 'work', 'nonexistent::'], expected_exit_code=1)
```

Reply:
is there some way to assert that the right issue is raised?

Reply:
I don't think so, as last time Ashwin suggested just put `expected_exit_code` to be 1.